spec/docs/participant_declarations_spec.rb

```diff
@@ -88,6 +88,8 @@
           RecordParticipantEvent.call(HashWithIndifferentAccess.new({ lead_provider: lead_provider, raw_event: params.to_json }).merge(params))
         end
 
+        schema "$ref": "#/components/schemas/id"
```

Comment:
Does this work in swagger?

I thought we didn't have anything like that set up (and it is a little bit of a faff to do)

Reply:
Well, apparently it does

Reply:
It seems it does, it'd based on the return type we already had in place for failures. I've updated the PR description with a screenshot of swagger so you can see it documenting the returned json (as well as a real response)